webview.h

```diff
@@ -900,6 +905,9 @@ class win32_edge_engine {
       m_window = CreateWindowW(L"webview", L"", WS_OVERLAPPEDWINDOW,
                                CW_USEDEFAULT, CW_USEDEFAULT, 640, 480, nullptr,
                                nullptr, GetModuleHandle(nullptr), nullptr);
+      if (m_window == nullptr) {
+        return;
```

Comment:
It may be appropriate to free `icon` using [DeleteObject](https://docs.microsoft.com/en-us/windows/desktop/api/wingdi/nf-wingdi-deleteobject) on failure (ugh, C-style resource management).

The result of [RegisterClassExW](https://docs.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-registerclassexw) might have the same issue and it might be appropriate to call [UnregisterClassW](https://docs.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-unregisterclassw) on failure.

While Windows cleans up all of this stuff when the process exits, a potential consequence of the new code is that it allows the process to keep running with these resources loaded into memory until the process exits.

For all I know that could be how it behaved even before these changes.